recipes/cminpack/bld.bat

```diff
@@ -0,0 +1,18 @@
+mkdir build
+cd build
+
+set CMAKE_BUILD_TYPE=RelWithDebInfo
```

Comment:
why not a Release build ?

Reply:
It's so small, adding debug info to it may come in handy when needing to debug things inside it (one of the advantages of using CMinpack instead of the one in Fortran is the ability to debug it better). The overhead is just a few kbytes (no runtime overhead).